### ISSUE TITLE
Deprecation warning in .107

### ISSUE THREAD

**Issue by arsaboo** [twrecked/lovelace-hass-aarlo#24]
Seeing the following error in .107. 

```bash
Log Details (WARNING)
Logger: homeassistant.components.camera
Source: components/camera/__init__.py:545
Integration: Camera (documentation, issues)
First occured: 6:57:07 PM (6 occurences)
Last logged: 7:37:50 PM

The websocket command 'camera_thumbnail' has been deprecated.
```

**Comment by arsaboo** [comment]
Looks like it is related to https://github.com/home-assistant/core/pull/32473

**Comment by twrecked** [comment]
Thanks for the heads up, I'll take a look.

**Comment by twrecked** [comment]
Fixed by #26 and in release https://github.com/twrecked/lovelace-hass-aarlo/releases/tag/v0.0.15
